mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala

```diff
@@ -228,6 +229,30 @@ object MLUtils extends Logging {
   }
 
   /**
+    * Version of [[labelKFold()]] taking a PairRDD with labels.
+    */
+  @Since("2.0.0")
```

Comment:
(This can't be since 2.0.0)
Let me comment on the change itself in the JIRA again. I understand the purpose now, but this is not about labels actually. We'd have to change the name and this wouldn't go in .mllib.